rules/do-you-know-how-to-manage-objections/rule.md

```diff
@@ -8,31 +8,74 @@ authors:
     url: https://ssw.com.au/people/adam-cogan
   - title: Ulysses Maclaren
     url: https://ssw.com.au/people/ulysses-maclaren
+  - title: Rob Thomlinson
+    url: https://www.ssw.com.au/people/RobThomlinson
 related: []
 redirects: []
 created: 2012-08-30T13:03:32.000Z
 archivedreason: null
 guid: d15ae8df-e369-44f9-a307-b48952431d7d
 ---
 
-When attempting to sell a solution to a potential client, you will invariably come up against some objections. It is essential that you are prepared to handle these objections so the client is confident in your skills and has no reservations about choosing you over someone else. The main reason clients raise objections is because they have concerns about your experience "fit" with their needs.
+Hearing “No” from a customer or manager can feel like a dead end, but it can be an opportunity to refine your approach. A rejection doesn’t mean they don’t want your solution. It might just mean they don’t yet see the value, have concerns, or need a different perspective.
```

Comment:
Get rid of the custom " and ' characters. This is a common issue when you copy/paste from Word (or...*other* sources)